src/components/tasks/card/index.tsx

```diff
@@ -392,7 +408,7 @@ const Card: FC<Props> = ({
                         <span
                             className={classNames.cardTitle}
                             contentEditable={shouldEdit}
-                            onKeyPress={(e) => handleChange(e, 'title')}
+                            onKeyDown={(e) => handleChange(e, 'title')}
```

Comment:
Hey @fakhruddinkw why you change from 
onKeyPress to onKeyDown here

Reply:
onKeyPress is deprecated. Its use is not recommended. And onKeyDown does the same job @shubhamsigdar1